sw/cheri/tests/plic_tests.hh

```diff
@@ -99,6 +105,11 @@ struct PlicTest {
       uart->interrupt_enable(uartMap[i].id);
       uart->interruptTest = ip_irq_id;
       wfi();
+      if (!irq_fired) {
+        error_count++;
+        log(static_cast<PLIC::Interrupts>(0), 0, true);
+      }
+      irq_fired = false;
```

Comment:
This clean up code is common to all of the IP blocks; I think it'd be safer to move it into the wfi() function itself, please, so that it remains present and correct for all IP blocks. Also, may I suggest that wfi() could perhaps be renamed so that it doesn't look quite so like a compiler intrinsic/assembler instruction; the casual reader may not spot the member function.